packages/avatars/src/elements/StatusIndicator.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { forwardRef, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { useText } from '@zendeskgarden/react-theming';
+import ClockIcon12 from '@zendeskgarden/svg-icons/src/12/clock-stroke.svg';
+import ClockIcon16 from '@zendeskgarden/svg-icons/src/16/clock-stroke.svg';
+import ArrowLeftIcon12 from '@zendeskgarden/svg-icons/src/12/arrow-left-sm-stroke.svg';
+import ArrowLeftIcon16 from '@zendeskgarden/svg-icons/src/16/arrow-left-sm-stroke.svg';
+
+import { IStatusIndicatorProps, STATUS } from '../types';
+import {
+  StyledStandaloneStatusIndicator,
+  StyledStandaloneStatusCaption,
+  StyledStandaloneStatus
+} from '../styled';
+
+/**
+ * @extends HTMLAttributes<HTMLElement>
+ */
+const StatusIndicatorComponent = forwardRef<HTMLElement, IStatusIndicatorProps>(
+  ({ children, type, isCompact, ...props }, ref) => {
+    let ClockIcon = ClockIcon16;
+    let ArrowLeftIcon = ArrowLeftIcon16;
+
+    if (isCompact) {
+      ClockIcon = ClockIcon12;
+      ArrowLeftIcon = ArrowLeftIcon12;
+    }
+
+    const defaultLabel = useMemo(() => ['status'].concat(type).join(': '), [type]);
+    const ariaLabel = useText(StatusIndicatorComponent, props, 'aria-label', defaultLabel);
+
+    return (
+      <StyledStandaloneStatus role="status" ref={ref} {...props}>
+        <StyledStandaloneStatusIndicator
+          role="img"
+          type={type}
+          isCompact={isCompact}
+          aria-label={ariaLabel}
+        >
+          {type === 'away' ? <ClockIcon data-icon-status={type} aria-hidden="true" /> : null}
+          {type === 'transfers' ? (
+            <ArrowLeftIcon data-icon-status={type} aria-hidden="true" />
+          ) : null}
+        </StyledStandaloneStatusIndicator>
+        {children && <StyledStandaloneStatusCaption>{children}</StyledStandaloneStatusCaption>}
+      </StyledStandaloneStatus>
+    );
+  }
+);
+
+StatusIndicatorComponent.displayName = 'StatusIndicator';
+
+StatusIndicatorComponent.propTypes = {
+  type: PropTypes.oneOf(STATUS).isRequired,
+  isCompact: PropTypes.bool
+};
+
+StatusIndicatorComponent.defaultProps = {
+  isCompact: false
```

Comment:
There should be a default specified for type. Please compare with https://github.com/zendeskgarden/react-components/blob/aeb2f8688ddeebca0bbfdd46ec234fdb101970e4/packages/buttons/src/elements/Button.tsx#L65 and other similar Garden prop constructs.

Reply:
If `type` is `required`, would it need a default value here?

Reply:
It wouldn't need to, but as a rule Garden provides a reasonable default. Let's do the same here.

Reply:
I suppose https://zendeskgarden.github.io/react-components/?path=/docs/packages-notifications-alert--alert could be a counter-argument. So let's go with whatever you think is best, @Francois-Esquire.

Reply:
Sure - @james-sann any thoughts on default status type? `("available" | "away" | "transfers" | "offline")`

Reply:
Agree with @jzempel on offline as default

Reply:
@Francois-Esquire moving error convo over here... I asked what happens when a browser is open and our network is done, or we can't get the data of the agent status. Do we ever see an error loading status? Or whats our fallback here?

Reply:
@abbelco typically in this case it would be up to the consumer to handle this. For the `StatusIndicator` component it will default to the `offline` status that we agreed to set as the default.

Reply:
Cool thanks for explaining!

Reply:
@Francois-Esquire Agree with the others. "Offline" to be the default